_posts/2018-03-09-starter-kit.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Starter Kit - the turn-key template for your own pages
+author: pitti
+date: 2018-03-09
+category: tutorial
+tags: cockpit starter-kit
+slug: cockpit-starter-kit
+comments: true
+---
+
+## The bare minimum
+
+[Cockpit's API](http://cockpit-project.org/guide/latest/development.html) makes it easy to create your own pages (or
+"plugins" if you will) that appear in Cockpit's menu and interact with your system in any way you like. Our pet
+example is the [Pinger](https://github.com/cockpit-project/cockpit/tree/master/examples/pinger) which is just the bare
+minimum: a [HTML file](https://github.com/cockpit-project/cockpit/blob/master/examples/pinger/ping.html) with a form to
+enter an IP, a small [piece of JavaScript](https://github.com/cockpit-project/cockpit/blob/master/examples/pinger/pinger.js)
+to call the `ping` Linux command  through Cockpit [spawn()](http://cockpit-project.org/guide/latest/cockpit-spawn.html)
+and capture its output; and a
+[manifest file](https://github.com/cockpit-project/cockpit/blob/master/examples/pinger/manifest.json) which tells
+cockpit how to add it to the menu and where the entry point is.
+
+There is a rather old [blog post](http://cockpit-project.org/blog/creating-plugins-for-the-cockpit-user-interface.html)
+which explains the Pinger example in detail. Cockpit changed its visual design quite dramatically since then, Pinger's
+JavaScript got split into a separate file and does not use jQuery any more, but aside from these details that post is
+still generally applicable.
+
+## Requirements for real projects
+
+Pinger is great for explaining and understanding the gist of how Cockpit works.  But an actual production-ready requires
```

Comment:
"production-ready [hamster] requires" -> production ready project?

Reply:
"hamster" was correct indeed :-P fixed, thanks for spotting!